pkg/build/controller/build/build_controller.go

```diff
@@ -1534,8 +1604,14 @@ func (bc *BuildController) handleBuildConfigError(err error, key interface{}) {
 }
 
 // createBuildCAConfigMap creates a ConfigMap containing certificate authorities used by the build pod.
-func (bc *BuildController) createBuildCAConfigMap(build *buildv1.Build, buildPod *corev1.Pod, update *buildUpdate) (*buildUpdate, error) {
-	configMapSpec := bc.createBuildCAConfigMapSpec(build, buildPod)
+func (bc *BuildController) createBuildCAConfigMap(build *buildv1.Build, buildPod *corev1.Pod, update *buildUpdate, additionalCas map[string]string) (*buildUpdate, error) {
+	configMapSpec, err := bc.createBuildCAConfigMapSpec(build, buildPod, additionalCas)
+	if err != nil {
+		bc.recorder.Eventf(build, corev1.EventTypeWarning, "FailedCreate", "Error creating build certificate authority configMap: %v", err)
+		update.setReason("CannotCreateCAConfigMap")
```

Comment:
Should this be a const?

Reply:
not as likely now that creating the spec doesn't error.